src/cryptonote_core/service_node_rules.h

```diff
@@ -8,7 +8,7 @@ namespace service_nodes {
   constexpr size_t PULSE_QUORUM_ENTROPY_LAG             = 21; // How many blocks back from the tip of the Blockchain to source entropy for the Pulse quorums.
   constexpr size_t PULSE_QUORUM_NUM_VALIDATORS          = 11;
   constexpr size_t PULSE_QUORUM_SIZE                    = PULSE_QUORUM_NUM_VALIDATORS + 1 /*Leader*/;
-  constexpr size_t PULSE_MIN_SERVICE_NODES              = 20;
+  constexpr size_t PULSE_MIN_SERVICE_NODES              = 50;
```

Comment:
Can we override this for testnet/fakenet?  I think it can just use `PULSE_QUORUM_SIZE`.

Reply:
I've made this parameterizable with a function that takes nettype